docs/contributors/getting-started.md

```diff
@@ -34,6 +34,13 @@ Once Docker is installed and running, run this script to install WordPress, and
 npm run env install
 ```
 
+You will then also have to build your local WordPress install by running
```

Comment:
It's odd that you should have to do this. `env install`, or a related command, should be handling any such needs automatically, IIRC. For instance, running `env update` pulls the latest changes and builds anew. I wonder if anything regressed.

Reply:
Yeah, the fact that Noah was running into the same problem seemed to indicate to me that something might've regressed...

Reply:
Right, I originally figured out what command to run because I looked through the code and saw that it was supposed to be running the npm commands within the WordPress repo. So it is probably worth fixing in the script if we want to keep this package working!